### PR TITLE
Glimpse: Don't use SEND_MULTIPLE if we have a single Uri

### PR DESCRIPTION
**Pull request by luca020400**
Change-Id: Ia4f99ed8fb7a30355f9910c66c808ba091693ca6

### CODE REVIEW

File: app/src/main/java/org/lineageos/glimpse/ext/Intent.kt [hunk 1]
```diff
@@ -9,25 +9,38 @@ import android.content.Intent
...
-        medias.map { it.externalContentUri }.toCollection(ArrayList())
-    )
-    type = when {
-        medias.all { it.mediaType == IMAGE } -> "image/*"
-        medias.all { it.mediaType == VIDEO } -> "video/*"
-        else -> {
-            putExtra(Intent.EXTRA_MIME_TYPES, arrayOf("image/*", "video/*"))
-            "*/*"
+@Suppress("UnusedReceiverParameter")
+fun KClass<Intent>.buildShareIntent(vararg medias: Media) = Intent().apply {
```
Comment (luca020400) [inline app/src/main/java/org/lineageos/glimpse/ext/Intent.kt]: Can't those be static functions?

Comment (SebaUbuntu) [reply app/src/main/java/org/lineageos/glimpse/ext/Intent.kt]: No static extensions in Kotlin, closest thing is companion object extensions, but `<class>.Companion` must exist in the original class, which doesn't for Intent

Comment (SebaUbuntu) [reply app/src/main/java/org/lineageos/glimpse/ext/Intent.kt]: Could be detached from Intent class, but I don't really like global static extensions

**Comment by luca020400** [comment]
+1

**Pull request merged**